### ISSUE TITLE
Levelz incompatability?

### ISSUE THREAD

**Issue by AlisterGontineac** [Draylar/go-fish#49]
https://paste.ee/p/O4rSW

**Comment by sweetsosweet** [comment]
I will add on to this as I posted in LevelZ github, but was mentioned the incompatibility was here?

Both load on their own.

Here's my mod test list:

- cloth-api-2.0.54
- cloth-config-5.0.38-fabric
- fabric-api-0.39.2+1.17
- go-fish-1.3.0-beta-1.17.1
- levelz-1.0.7

(also tried later with fabric-api-0.40.1+1.17)

Here's my log: https://gist.github.com/sweetsosweet/d22154f3dd934b79279323fe6a973295

I don't know any coding, but it was mentioned the following:

"Go Fish uses an Overwrite inside the FishingRodItem class on the use method which makes it incompatible with all other mods which try to inject stuff into the same method. It doesn't need to be an Overwrite."

**Comment by DanMeadWasTaken** [comment]
Well, I came here to make the same complaint, but seems he pretty much summed everything up.

**Comment by Draylar** [comment]
Will be fixed in 1.3.1* on 1.17.1 and 1.18.1, thanks for the report.




...
\* *My solution isn't fantastic because I'm not sure how to modify a new instance being passed to a method like this - if anybody knows of a way to do this, please let me know! I've switched to a redirect for now, which will fix the issue with LevelZ, but will most likely conflict again in the future.*
```java
world.spawnEntity(new FishingBobberEntity(user, world, k, I));
```
